### ISSUE TITLE
Multiple identities feature request

### ISSUE THREAD

**Issue by Nic13Gamer** [technomancy-dev/00#17]
In Resend, we can add multiple domains to send from. It would be very nice in 00 if we could send from multiple domains also, so it's possible to have only one 00 container running and send emails from various domains. Maybe this could be a nice feature for the Pro version.

**Comment by liltechnomancer** [comment]
Awesome thanks for the suggesting! That would be great to add.

**Comment by CestDiego** [comment]
Indeed, I'd love to have this too since I'm managing many projects on my end :) Looking forward to the pro version, great job!

**Comment by Qodestackr** [comment]
Can I use React Email with this?

**Comment by liltechnomancer** [comment]
> Can I use React Email with this?

Yes, you will use it on your side and send the HTML it renders.
